src/ml/neural_net/style_transfer/mps_style_transfer_vgg_16_block_1_node.h

```diff
@@ -0,0 +1,27 @@
+#import <Accelerate/Accelerate.h>
```

Comment:
You should have a copyright header here and at the start of every other new file.

Reply:
Thank you!